core/trino-spi/src/main/java/io/trino/spi/block/RowBlockBuilder.java

```diff
@@ -422,4 +441,11 @@ private Block nullRle(int length)
         RowBlock nullRowBlock = createRowBlockInternal(1, new boolean[] {true}, fieldBlocks);
         return RunLengthEncodedBlock.create(nullRowBlock, length);
     }
+
+    private static void checkArgument(boolean expression, String errorMessage, Object... messageArgs)
```

Comment:
If we don’t have some version of Preconditions / checkArgument in scope within the SPI module, then we should just inline this at the call site